### PR TITLE
Events beta docs update

### PR DESCRIPTION
**Pull request by ardaguclu**
Enhancement issue: https://github.com/kubernetes/enhancements/issues/1440

Kubernetes PRs:
- https://github.com/kubernetes/kubernetes/pull/113819
- https://github.com/kubernetes/kubernetes/pull/110007
- https://github.com/kubernetes/kubernetes/pull/110766
- https://github.com/kubernetes/kubernetes/pull/111855

### CODE REVIEW

**Comment by krol3** [comment]
CC @mickeyboxell

**Comment by ardaguclu** [comment]
This is my first change on kubernetes/website and I'm not sure changing kubectl-commands in here https://github.com/kubernetes/website/blob/ed0578b98e4756eb17b4b29c6d6543230066731e/static/docs/reference/generated/kubectl/kubectl-commands.html or is this html autogenerated?

Other than that, since this PR promotion of events command from alpha to beta. I added it into command list. 

/cc @soltysh

**Review by soltysh — APPROVED** [review]
/lgtm
from tech pov
@ardaguclu you need approval from sig-docs

File: content/en/docs/reference/kubectl/_index.md [hunk 1]
```diff
@@ -137,6 +137,7 @@ Operation       | Syntax    |       Description
 `drain`    | `kubectl drain NODE [options]` | Drain node in preparation for maintenance.
 `edit`        | <code>kubectl edit (-f FILENAME &#124; TYPE NAME &#124; TYPE/NAME) [flags]</code> | Edit and update the definition of one or more resources on the server by using the default editor.
 `exec`        | `kubectl exec POD [-c CONTAINER] [-i] [-t] [flags] [-- COMMAND [args...]]` | Execute a command against a container in a pod.
+`events`      | `kubectl events` | List events
```
Comment (tengqm) [inline content/en/docs/reference/kubectl/_index.md:140]: this command should appear before `exec`?

**Comment by krol3** [comment]
Hi @ardaguclu! This PR needs a doc review by **Mon Nov 28th** to get this into the release. Please reach out to required SIGs to get their review. Thank you!

**Comment by ardaguclu** [comment]
@tengqm thanks for review. Updated, could you PTAL?

**Review by sftim — COMMENTED** [review]
Change LGTM

/lgtm

**Comment by krol3** [comment]
@soltysh  can you approve this PR ?

**Comment by soltysh** [comment]
> @soltysh can you approve this PR ?

approval has to be from sig-docs, @sftim can you make the honors, this is :+1: from sig-cli pov

**Pull request merged**